Gemfile

```diff
@@ -2,7 +2,7 @@ source ENV['GEM_SOURCE'] || "https://rubygems.org"
 
 gemspec
 
-group :release do
-  gem 'faraday-retry', require: false
-  gem 'github_changelog_generator', require: false
+group :release, optional: true do
```

Comment:
IF you make it optional, you can drop this:
https://github.com/voxpupuli/beaker-openstack/blob/d44ba424ca0919d70e3d359223a02e557c68b980/.github/workflows/test.yml#L23-L24

And I've grown fond of https://github.com/theforeman/actions#gem-test as a way to test gems on all the Ruby versions that are supported according to the gemspec. Though I should update https://github.com/ekohl/ruby-version to add Ruby 3.3.

Reply:
made it optional. we should talk about unifying our CI setup for all gems and maybe use the shared workflow.